### PR TITLE
show invalid fingerprint in exception

### PR DESCRIPTION
**Pull request by t-8ch**

### CODE REVIEW

File: urllib3/util/ssl_.py [hunk 1]
```diff
@@ -123,7 +123,8 @@ def assert_fingerprint(cert, fingerprint):
     digest_length = len(fingerprint)
     hashfunc = HASHFUNC_MAP.get(digest_length)
     if not hashfunc:
-        raise SSLError('Fingerprint is of invalid length.')
+        raise SSLError(
+            'Fingerprint {0} is of invalid length.'.format(fingerprint))
```
Comment (shazow) [inline urllib3/util/ssl_.py]: For consistency with other message styles, how about we do: `'Fingerprint is of invalid length: {}'`? (Easier to split/parse if you need to this way.)

Comment (shazow) [inline urllib3/util/ssl_.py]: Or maybe grammatically better yet, "Invalid fingerprint length: {}"

Comment (t-8ch) [inline urllib3/util/ssl_.py]: I wanted to show the fingerprint, not necessarily the length itself.

**Comment by shazow** [comment]
Perfect, thanks!

**Pull request merged**